### DISCUSSION TITLE
No 'Add metabolizing enzyme... (Database query)' option available after specifying the path

### DISCUSSION THREAD

**Discussion by JoycevdHeijden** [Open-Systems-Pharmacology/Forum#1359] [Questions & Problems] [answered]
Dear OSP community, I'm new to figuring out PK-Sim and I'm following the Tutorials from https://www.open-systems-pharmacology.org/. I've been trying to open the protein expression database in PK-Sim to view the 'Add metabolizing enzyme... (Database query)' option. I've downloaded the 'Gene expression database (human)' from the site https://github.com/Open-Systems-Pharmacology/Suite/releases/tag/v11.0. I've specified the path to the species-specific (human) expression database in the utilities > Options > Application tab. Though, the database query option isn't available. I've restarted the software as well to check if this could have caused the issue. Why is the option not showing? Many thanks in advance!

**Comment by AndreDlm** [comment, marked as answer]
Hi Joyce, the protein expression has a bit changed in PK-Sim v11 and can now be found as separate Building Block. You can add a new protein via the context menu of that Building Block:
![image](https://user-images.githubusercontent.com/26541603/197577435-22aaaf0b-85cf-4533-867f-25479c39c103.png)
In the menu that pops up you can either type in the enzyme name or select one from the list, e.g.:
![image](https://user-images.githubusercontent.com/26541603/197577662-9cc6a45f-a476-414e-9d60-471d18501586.png)
This creates an (empty) placeholder for the enzyme without any tissue expression. You can then either add tissue expression manually by overwriting the values for relative expression or initial concentration in the list below or you click on `Database Query` to load it from the (human) expression database:
![image](https://user-images.githubusercontent.com/26541603/197578131-2ff9fb43-d3e1-40c9-bcca-841f30402ace.png)

You can find more information in the manual:
https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-expression-profile#definition-of-new-expression-profile-in-pk-sim-r

I hope this helps,
André

**Reply by JoycevdHeijden** [reply]
Thank you so much for the quick answer, it's clear now!
